uicomponent-compose/core/src/main/java/io/github/droidkaigi/feeder/core/language/LanguageTitle.kt

```diff
@@ -0,0 +1,12 @@
+package io.github.droidkaigi.feeder.core.language
+
+import android.content.Context
+import io.github.droidkaigi.feeder.Lang
+import io.github.droidkaigi.feeder.core.R
+
+fun Lang?.getTitle(context: Context): String = when (this) {
```

Comment:
Passing a context to this will probably make our app a bit more complicated. 
How about using a composable function, LocalContext.current?
This is only for refactoring. So there is no need to do this for this PR. 😄